docs/external-listener/index.md

```diff
@@ -59,17 +59,15 @@ To configure an external listener that uses the LoadBalancer access method, comp
           accessMethod: LoadBalancer
     ```
 
-1. Configure an *ingress controller* in the `ingressController` field of the `KafkaCluster` custom resource. For example:
+1. Set the ingress controller. The ingress controllers that are currently supported for load balancing are:
 
-    ```yaml
-    spec:
-      ingressController: "envoy"
-    ```
+    - `envoy`: uses Envoy proxy as an ingress.
+    - `istioingress`: uses Istio proxy gateway as an ingress. Istio ingress is the default controller for Kafka clusters provisioned with [Supertubes](/docs/supertubes/overview/), since those clusters run inside an Istio mesh.
 
-    The ingress controllers that are currently supported are:
+    Configure the ingress controller you want to use:
 
-    -  envoy: uses Envoy Proxy as an ingress controller.
-    -  istioingress: uses Istio Gateway as an ingress controller. This is the default controller for Kafka clusters provisioned with [Supertubes](/docs/supertubes/overview/), since those clusters run inside an Istio mesh.
+    - To use Envoy, set the `ingressController` field in the `KafkaCluster` custom resource to `envoy`. For an example, see [https://github.com/banzaicloud/koperator/blob/master/config/samples/banzaicloud_v1beta1_kafkacluster.yaml](https://github.com/banzaicloud/koperator/blob/master/config/samples/banzaicloud_v1beta1_kafkacluster.yaml)
+    - To use Istio ingress controller set the `ingressController` field to `istioingress`. Istio operator v2 is supported from Koperator version 0.21.0+. Because of this, you must also configure the `istioControlPlane` field in the `KafkaCluster` custom resource. `istioControlPlane` is a reference to the IstioControlPlane resource. For an example, see [https://github.com/banzaicloud/koperator/blob/master/config/samples/kafkacluster-with-istio.yaml](https://github.com/banzaicloud/koperator/blob/master/config/samples/kafkacluster-with-istio.yaml)
```

Comment:
```suggestion
    - To use Istio ingress controller set the `ingressController` field to `istioingress`. Istio operator v2 is supported from Koperator version 0.21.0+. Because of this, you must also configure the `istioControlPlane` field in the `KafkaCluster` custom resource. `istioControlPlane` is a reference to the IstioControlPlane resource. For an example, [see](https://github.com/banzaicloud/koperator/blob/master/config/samples/kafkacluster-with-istio.yaml)
```